code.gs

```diff
@@ -1588,6 +1587,7 @@ function getEventsOnDate (eventDate, calendarId) {
 }
 
 /**
+ * <div style="clear:both"></div>
```

Comment:
Makes me wonder why markdown formatting uses html tag...

Reply:
Markdown is just a transpiler to html (and allows inline html) anyway, and from what I could see the standard jsdoc style (for some crazy reason) has the descriptions "clear: none" after the function-name headings, yet a huge right-side padding for the heading. In most cases that is not visible because the function-name/retvals are wide enough to force a linebreak in the formatting anyway. It just happens that the `main()` function has a short name and no retval, so the description ends up being some squished semi-lines far over on the right of it. This ugly clear-both div bugs me, but was the only way I found to fix the formatting without digging into the jsdoc-stylesheet itself (which I really am not in the mood for). If anyone finds a less ugly workaround (or wants to try overriding the stylesheet - perhaps in the conf-file) please add it.

Reply:
Oh, I see, thanks for explaining it for me.